### ISSUE TITLE
`scipy.linalg.pinv2` not available in scipy `1.9`

### ISSUE THREAD

**Issue by mmaelicke** [GeoStat-Framework/GSTools#246]
Hey Sebastian,
I am running some docker containers with always the latest gstools version in it. Now, the import of gstools fails.
Here is the relevant part of the traceback:

```
File "/usr/local/lib/python3.10/site-packages/skgstat/interfaces/gstools.py", line 151, in skgstat_to_krige
    import gstools as gs
File "/usr/local/lib/python3.10/site-packages/gstools/__init__.py", line 129, in <module>
    from gstools import (
File "/usr/local/lib/python3.10/site-packages/gstools/field/__init__.py", line 33, in <module>
    from gstools.field.cond_srf import CondSRF
File "/usr/local/lib/python3.10/site-packages/gstools/field/cond_srf.py", line 17, in <module>
    from gstools.krige import Krige
File "/usr/local/lib/python3.10/site-packages/gstools/krige/__init__.py", line 20, in <module>
    from gstools.krige.base import Krige
File "/usr/local/lib/python3.10/site-packages/gstools/krige/base.py", line 38, in <module>
    P_INV = {"pinv": spl.pinv, "pinv2": spl.pinv2, "pinvh": spl.pinvh}
```

I could verify, that `scipy==1.9.0` in fact has no `scipy.linalg.pinv2` method, while `scipy==1.8.0` imports it just fine. The whole practice was run in a Python 3.10 environment.
I solved the issue by installing scipy==1.8.0 first and then gstools. I am also not sure if the `pinv2` got moved or removed and if therefore any action on gstools-side is necessary.

**Comment by mmaelicke** [comment]
https://github.com/scipy/scipy/issues/15737  - Seems like `pinv2` should be deprecated and/or removed without a deprecation warning?

**Comment by LSchueler** [comment]
Hi Mirko,
thanks for reporting this! It should be resolved with PR #247 .

**Comment by MuellerSeb** [comment]
Also see: https://github.com/GeoStat-Framework/PyKrige/issues/236
